test/integration/handlers/callback-response.spec.ts

```diff
@@ -1,6 +1,9 @@
 import * as zlib from 'zlib';
 import { getLocal } from "../../..";
 import { expect, fetch, isNode, isWeb, headersToObject } from "../../test-utils";
+import * as http from 'node:http';
```

Comment:
The tests are failing because of this - this spec runs in browsers too, so you can't use Node modules like this.

You _could_ make this test node-only, and then this would work if you import this directly as `http` (the tests are already configured to exclude that in browser builds) but it would be better to write this so it will pass in both node & browsers.

I think you can just use streams directly without `http.request` to test this equally well, in the same way we do testing in https://github.com/httptoolkit/mockttp/blob/main/test/integration/handlers/stream-response.spec.ts.

Reply:
Oops :) The one part of the dev environment that I hadn't setup happens to be exactly where I created a test failure. I picked `node:http` instinctively because that's what I was going to use in my planned use case for this feature. I'll try to figure this out today.